src/Symfony/Component/Console/Formatter/OutputFormatterStyle.php

```diff
@@ -29,6 +29,7 @@ class OutputFormatterStyle implements OutputFormatterStyleInterface
         'magenta' => array('set' => 35, 'unset' => 39),
         'cyan' => array('set' => 36, 'unset' => 39),
         'white' => array('set' => 37, 'unset' => 39),
+        'default' => array('set' => 39, 'unset' => 39),
```

Comment:
I think `default` is the right term (it depends on the user's settings whether this is transparent, black, white, or something else).